### PR TITLE
Fix of Issue #2470

### PR DESCRIPTION
**Pull request by luckyman000786**
As suggested by @mbj36

### CODE REVIEW

File: app/views/loans/viewloanaccountdetails.html [hunk 1]
```diff
@@ -897,8 +897,8 @@ <h3 class="bolder">{{'label.heading.confirm' | translate}}</h3>
                             <div class="form-group">
                                 <div class="col-sm-12 col-md-12">
                                     <textarea rows="1" class="form-control marginbottom0px" id="textarea"
-                                              placeholder="{{'label.input.enternote' | translate}}" ng-model="formData.note"></textarea>
-
+                                              placeholder="{{'label.input.enternote' | translate}}" ng-model="formData.note" style="min-width: 1050px; max-width: 1050px; "></textarea>
```
Comment (mbj36) [inline app/views/loans/viewloanaccountdetails.html]: What is the use of styling here ?

Comment (luckyman000786) [reply app/views/loans/viewloanaccountdetails.html]: If we don't use max-width here, the text box can go outside the div container

Comment (mbj36) [reply app/views/loans/viewloanaccountdetails.html]: Did you tried in local ? I don't think it will go outside, if it goes please post the screenshot here

Comment (luckyman000786) [reply app/views/loans/viewloanaccountdetails.html]: capture_2017_10_04_21_20_33_708
Here you can see

Comment (mbj36) [reply app/views/loans/viewloanaccountdetails.html]: I see you have added some style there in Inspector! 

This is what i see pulling your code and removing the styling in `text-area`
selection_066

Comment (mbj36) [reply app/views/loans/viewloanaccountdetails.html]: So probably the fix would be removing that styling from `text-area`

Comment (luckyman000786) [reply app/views/loans/viewloanaccountdetails.html]: No, I haven't added the these style, these are default by firefox (the height and width of text-area).  When increasing the width, it goes outside of <div class="card">

Comment (luckyman000786) [reply app/views/loans/viewloanaccountdetails.html]: capture
With my fix user is restricted to 1050 px width and user can't extend it beyond the div element

Comment (luckyman000786) [reply app/views/loans/viewloanaccountdetails.html]: capture2
And without my fix user can extend it beyond the div element and then cannot resize it

Comment (mbj36) [reply app/views/loans/viewloanaccountdetails.html]: Thanks @luckyman000786

**Pull request merged**